apps/web/src/utils/file-upload.ts

```diff
@@ -11,11 +11,14 @@ export function createFileUploadHandler(onSuccess: (file: File) => void) {
     onSuccess,
   }));
 
-  return (event: { target: HTMLInputElement }) => {
-    const [file] = event.target.files ?? [];
+  return [
+    (event: { target: HTMLInputElement }) => {
+      const [file] = event.target.files ?? [];
 
-    if (file) {
-      fileUpload.mutate(file);
-    }
-  };
+      if (file) {
+        fileUpload.mutateAsync(file);
```

Comment:
If using `mutateAsync`, we should either await or explicitly discard the promise (with `void ...`).

Reply:
It works fine with `mutate` on my end.